### PR TITLE
add callbacks on tree

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 - 为 Notification 和 Message 的不同 type 添加独立的调用方法
 - 为 Message Box 和 Dialog 添加 lockScroll 属性，用于定义是否在弹框出现时将 body 滚动锁定
 - 新增 Input textarea 类型的 rows, autosize 属性
+- 为 Tree 添加 getCheckedNodes 方法和 node-click、select-change 回调
```
Comment (furybean) [inline CHANGELOG.md]: select-change?

Comment (Leopoldthecoder) [inline CHANGELOG.md]: 哦，check-change= =

**Comment by Leopoldthecoder** [comment]
已更新

**Comment by QingWei-Li** [comment]
approved

**Pull request merged**